v4/export/ir_impl_test.go

```diff
@@ -38,15 +50,54 @@ func (s *testIRImplSuite) TestRowIter(c *C) {
 	for i := 0; i < 100; i += 1 {
 		c.Assert(iter.HasNext(), IsTrue)
 	}
-	res := &simpleRowReceiver{}
+	res := newSimpleRowReceiver(1)
 	c.Assert(iter.Next(res), IsNil)
-	c.Assert(res.data, Equals, "1")
+	c.Assert(res.data, DeepEquals, []string{"1"})
 	c.Assert(iter.HasNext(), IsTrue)
 	c.Assert(iter.HasNext(), IsTrue)
 	c.Assert(iter.Next(res), IsNil)
-	c.Assert(res.data, Equals, "2")
+	c.Assert(res.data, DeepEquals, []string{"2"})
 	c.Assert(iter.HasNext(), IsTrue)
 	c.Assert(iter.Next(res), IsNil)
-	c.Assert(res.data, Equals, "3")
+	c.Assert(res.data, DeepEquals, []string{"3"})
 	c.Assert(iter.HasNext(), IsFalse)
 }
+
+func (s *testIRImplSuite) TestSizedRowIter(c *C) {
+	db, mock, err := sqlmock.New()
+	c.Assert(err, IsNil)
+	defer db.Close()
+
+	twentyBytes := repeat("x", 20)
```

Comment:
```suggestion
	twentyBytes := strings.Repeat("x", 20)
```

https://godoc.org/strings#Repeat